### ISSUE TITLE
First prototype - Server (ElasticSearch)

### ISSUE THREAD

**Issue by alecron** [IAAA-Lab/space-ql#15]
- [x] ES searches for the whole content file
- [x] ES searches for specific fields like title and description
- [x] ES can search for fields like the contact points or the type of metadata being served (service or dataset)
- [x] ES and the client work together to serve a dynamic experience for the user, who can navigate very intuitively through the data

**Comment by fjlopez** [comment]
> When ES is asked to analyze the whole content, it doesn't return any matches, so it doesn't seem to be the best choice. So, storing the whole content in the ES database and using jsonPath on the server doesn't look like the best option.

If you wish, I can repeat the failed task and provide you hints or solutions. Please, create a new issue with the steps to reproduce the problem.

**Comment by alecron** [comment]
> If you wish, I can repeat the failed task and provide you hints or solutions. Please, create a new issue with the steps to reproduce the problem.

I just found the problem. I stored the Json String into the "document"/content field, but it has scaped characters like quotes or  accent marks that makes words like "España" or "Básico" unfoundable. it is a bit tricky to find something with that format but i'll try to change the charset to see if it fixes the problem. 

I will also change the fuzziness of the search, so it doesn't look strictly for the word as it is written in the input.
